> Did you check if py-cryptography works as expected? There was a reason why openssl was built by spack in the past.

Did not (is there a simple way to do so?), but `UFS WM` fails with a `spack`-built `openssl`:

`ld: /apps/contrib/spack-stack/spack-stack-1.9.2rc2/envs/ue-oneapi-2024.1.0/install/oneapi/2024.2.1/curl-8.10.1-k3j33tn/lib/libcurl.so.4: undefined reference to SSL_get0_group_name@OPENSSL_3.2.0`

@RatkoVasic-NOAA can speak to this; kind of a Catch-22.

--- a/configs/sites/tier1/ursa/compilers.yaml
+++ b/configs/sites/tier1/ursa/compilers.yaml
@@ -31,3 +31,18 @@ compilers:
     modules: []
     environment: {}
     extra_rpaths: []
+
+- compiler:
+    spec: gcc@12.4.0
+    paths:
+      cc: /apps/spack-2024-12/linux-rocky9-x86_64/gcc-11.4.1/gcc-12.4.0-dsgnou52lpn2tus6mohdmcw5mjqmqrhj/bin/gcc
+      cxx: /apps/spack-2024-12/linux-rocky9-x86_64/gcc-11.4.1/gcc-12.4.0-dsgnou52lpn2tus6mohdmcw5mjqmqrhj/bin/g++
+      f77: /apps/spack-2024-12/linux-rocky9-x86_64/gcc-11.4.1/gcc-12.4.0-dsgnou52lpn2tus6mohdmcw5mjqmqrhj/bin/gfortran
+      fc: /apps/spack-2024-12/linux-rocky9-x86_64/gcc-11.4.1/gcc-12.4.0-dsgnou52lpn2tus6mohdmcw5mjqmqrhj/bin/gfortran
+    flags: {}
+    operating_system: rocky9
+    target: x86_64
+    modules:
+    - gcc/12.4.0
+    environment: {}
+    extra_rpaths: []

--- a/configs/sites/tier1/ursa/packages_gcc.yaml
+++ b/configs/sites/tier1/ursa/packages_gcc.yaml
@@ -1,6 +1,6 @@
 packages:
   all:
-    compiler:: [gcc@11.4.1]
+    compiler:: [gcc@12.4.0]
     providers:
       mpi:: [openmpi@4.1.6]
   mpi:
@@ -13,6 +13,5 @@ packages:
       - openmpi/4.1.6
   gcc-runtime:
     externals:
-    - spec: gcc-runtime@11.4.1%gcc@11.4.1
+    - spec: gcc-runtime@12.4.0%gcc@12.4.0
       prefix: /usr
-

